Python-Input.md

```diff
@@ -0,0 +1,33 @@
+# Python Input() Function
+
+Many a times, in a program we need some input from the user. Taking inputs from the user makes the program feel a bit interactive. In python, to take input from the user we have a function `input()`. Let's see some examples:
+
+1. When we just want to take the input:
+```python
+# This will just give a prompt without any message
+inp = input()
+```
+[Run Code](https://repl.it/CUqX/0)
```

Comment:
Don't think you would need a REPL snippet here. It does nothing when you run it. Except wait for an input.

@koustuvsinha @atjonathan @Rafase282 what do you guys think?